### PR TITLE
Fix 554

### PR DESCRIPTION
**Pull request by rmichaelis**
@jaykayone Can you please test the infobar in your Apple environment. ( scale and projection  hover as well as the position of the elements)
Thanks,

### CODE REVIEW

**Comment by rmichaelis** [comment]
mouse location element too large and info bar does not open on mobile device

**Comment by rmichaelis** [comment]
Ready to be reviewed

**Comment by rmichaelis** [comment]
No more comment about this one ?

File: geoportailv3/static/js/infobar/projectionselector.html [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+      </button>
+      <ul class="dropdown-menu" role="menu">
+        <li ng-repeat="projitem in ::ctrl.projectionOptions">
+          <a href ng-click="ctrl.switchProjection(projitem.value)" ng-bind-html="::projitem.label"></a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="mouse-coordinates" ngeo-control="ctrl.mousePositionControl" ngeo-control-map="ctrl.map"></div>
+</div>
```
Comment (elemoine) [inline geoportailv3/static/js/infobar/projectionselector.html]: You may want to fix the "No newline at end of file" error here.

**Comment by elemoine** [comment]
I'll let someone else review this as this is too UI-oriented for me :)

**Comment by petzlux** [comment]
fixes #316

**Comment by petzlux** [comment]
Can this be merged, can @pgiraud  please review ?

**Comment by pgiraud** [comment]
I will not be able to review it today. I hope tomorrow.
By the way, it's not mergeable. Needs a rebase.

**Comment by rmichaelis** [comment]
Rebase done

**Comment by pgiraud** [comment]
Ok, this looks good. Please merge.

**Pull request merged**